src/test/java/com/exasol/adapter/databricks/AdapterIT.java

```diff
@@ -65,13 +68,24 @@ void databricksMetadataAvailableInAdapterNotes() {
                                 containsString("\"type_text\":\"varchar(5)\""))));
     }
 
+    @ParameterizedTest
+    @EnumSource(AuthMode.class)
+    void databricksAuthentication(final AuthMode authMode) {
```

Comment:
Shouldn't be parameter `authMode` be used somewhere in the test body?

Reply:
It's used in line 77 for creating the VS:
```java
testSetup.exasol().createVirtualSchema(databricksSchema, emptyMap(), authMode);
```

Reply:
Oh, didn't see that.